packages/alias/package.json

```
@@ -14,6 +14,9 @@
     "node.js",
     "path alias"
   ],
+  "engines": {
+    "node": ">=22"
```

This isn't fully true: `module.register` exists in node [18.x and 20.x](https://nodejs.org/api/module.html#moduleregisterspecifier-parenturl-options)

```suggestion
    "node": ">=22 || ^20.6.0 || ^18.19.0"
```